camayoc/tests/qpc/api/v1/credentials/test_network_creds.py

```diff
@@ -132,13 +140,15 @@ def test_create_with_sshkey(shared_client, cleanup, isolated_filesystem):
     :steps: Send POST with necessary data to documented api endpoint.
     :expectedresults: A new network credential entry is created with the data.
     """
-    ssh_keyfile = Path(uuid4())
-    ssh_keyfile.touch()
+    sshkeyfile_name = utils.uuid4()
```

Comment:
since we are using pathlib, you could do something like:

```python
sshkeyfile_name = utils.uuid4()
sshkeyfile = Path('.') / f'{sshkeyfile_name}'
sshkeyfile.touch()
```

More info here https://docs.python.org/3.5/library/pathlib.html#operators

Reply:
These are great suggestions. Since I will likely have to tweak these lines once I implement a fix for Issue #338, I think I'll leave it as is for now and investigate these options while working on that fix :slightly_smiling_face: 

Thanks for the feedback!